src/components/vm/nics/nicAdd.tsx

```diff
@@ -211,14 +232,20 @@ export class AddNIC extends React.Component<AddNICProps, AddNICState> {
             } else {
                 source = this.state.networkSource;
             }
+            let backend = null;
+            if (this.state.networkType == "user" && vm.capabilities.interfaceBackends.includes("passt"))
+                backend = "passt";
             await virtXmlHotAdd(
                 vm,
                 "network",
                 {
                     mac: this.state.setNetworkMac ? this.state.networkMac : getRandomMac(vms),
                     model: this.state.networkModel,
                     type: this.state.networkType,
+                    backend: { type: backend },
                     source,
+                    portForward: this.state.networkType == "user" ? dialogPortForwardsToInterface(this.state.portForwards) : null,
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2331-66f0c2f5-20251104-070631-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)